Add "Enrich Domain with OSINT from Slack"

PhilippBehmer opened the pull request:
WWF "Enrich Domain with OSINT from Slack"

jmcadams-r7 commented on workflows/Enrich_Domain_with_OSINT_from_Slack/help.md:
```diff
@@ -0,0 +1,63 @@
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
```
Need instructions on configuring Slack

PhilippBehmer replied:
I added a link to our Slack App documentation and a description on how to change from direct msg to channel listener

jmcadams-r7 commented on workflows/Enrich_Domain_with_OSINT_from_Slack/Enrich_Domain_with_OSINT_from_Slack.icon:
```diff
@@ -0,0 +1,1232 @@
+            "nodeId": "ad6f309a-b15d-48a3-93ef-ea3bf0f1a143",
+            "name": "Summary",
+            "type": "artifact",
+            "continueOnFailure": false,
```
You'll need to set continueOnFailure to True, and/or add error messages in the flow. If the WF bombs out, we want the user to see that in slack with a link back to the failed job.

PhilippBehmer replied:
@jmcadams-r7 I implemented a check if one of the steps fails and ICON will send a Slack msg including the job url in case of a step failure.



Not

PhilippBehmer commented:
I implemented your change requests, updated the plugins, and also use cloud plugins now which run MUCH faster 🚀 

@jmcadams-r7 & @sengleson-r7 can you please review?

sengleson-r7 commented on workflows/Enrich_Domain_with_OSINT_from_Slack/help.md:
```diff
@@ -0,0 +1,67 @@
+
+# Key Features
+
+* ThreadCrowd URL Lookup
```
```suggestion
* Threat Crowd URL Lookup
```

PhilippBehmer replied:
Thank you! Fixed!

sengleson-r7 reviewed (changes requested):
I just imported and ran this, it was done before I could even open the thread! Awesome work Phil! One really small typo in the key features =)

sengleson-r7 commented on workflows/Enrich_Domain_with_OSINT_from_Slack/workflow.spec.yaml:
```diff
@@ -0,0 +1,25 @@
+status: []
+hub_tags:
+  use_cases: [data_enrichment, threat_detection_and_response]
+  keywords: [slack, chat, chatops, work_from_home]
```
```suggestion
  keywords: [slack, chat, chatops, work_from_home, cloud_enabled]
```

PhilippBehmer replied:
yep! Thanks for adding the tag!

sengleson-r7 reviewed (changes requested):
Sorry Phil! I just realized, this would be a cloud enabled workflow -- zero deployment required!!

jschipp-r7 commented:
Successfully validated

```
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 28.357ms
```
